src/components/pages/VehiclesDetails.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import './styles/details.css';
+
+const VehiclesDetails = () => {
```

Comment:
- [ ] According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/227380838-04f3b509-d36b-413d-86eb-65f2dfbbefa1.png)


Therefore, kindly add a reserve button to the motorcycles details pages, so your users can go to the reservations form directly from there.

![image](https://user-images.githubusercontent.com/57421823/227380800-3361a595-7d43-4196-ae1f-a002b3316e66.png)

Reply:
Added a reserve button on the motorcycles details pages